lib/app/routes/core.rb

```diff
@@ -143,6 +147,7 @@ def planned_languages
         def tracks
           ExercismWeb::Presenters::Tracks
```

Comment:
Hm. Actually, in terms of the project, the term _track_ is a very distinct thing: It is the exercises in a specific language on exercism. The difference between a _language_ and a _track_ is that a language exists independently of exercism, whereas a track is a specific implementation of a set of exercises in a given language... on exercism. If that makes sense.

Instead of having two methods that are kind of synonymous, but with kind of misleading names, let's fix the `def tracks` method so that it returns the actual tracks (the thing that `all_languages` is currently returning), and then get rid of the `all_languages` method.

You will need to refactor a little bit, and it will no longer make sense to have the class methods `active`, `upcoming`, and `planned`... it's better to simply do the `tracks.select(&:active?)` in the controller where it's needed.

Does that make sense?